docs/content/backup_restore.md

```diff
@@ -15,14 +15,39 @@ With time, this directory will grow, that's why a [backup expiration age](./conf
 was added. Every time you launch the command, at the end, it will be asked if you want to remove obsolete
 backup files (i.e. files that have passed their expiration date).
 
+<div class="standalone">
+
+```sh
+vendor/bin/db-tools backup
+```
+
+</div>
+<div class="symfony">
+
 ```sh
 console db-tools:backup
 ```
 
+</div>
+
 You can specify the behavior of the command with some options detailed below.
 
 ### Connection
 
+<div class="standalone">
```

Comment:
Doing that you loose tabs?

Reply:
yep, you only have your current selected flavor examples

Reply:
Does this happen often?